nemo/collections/tts/models/fastpitch.py

```diff
@@ -151,6 +153,38 @@ def tb_logger(self):
             self._tb_logger = tb_logger
         return self._tb_logger
 
+    @property
+    def normalizer(self):
+        if self._normalizer is not None:
+            return self._normalizer
+
+        if self.learn_alignment:
+            ds_class_name = self._cfg.train_ds.dataset._target_.split(".")[-1]
```

Comment:
Have you checked to see if this works with `from_pretrained`? I think `from_pretrained` deletes `self._cfg.train_ds` and validation

Reply:
Closed in discussion